### PR TITLE
Include transaction in database call which has been accidentally left out.

### PR DESCRIPTION
**Pull request by DamienLaw**

### CODE REVIEW

File: src/Dommel/DommelMapper.Insert.cs [hunk 1]
```diff
@@ -1,10 +1,10 @@
+using Dapper;
```
Comment (henkmollema) [inline src/Dommel/DommelMapper.Insert.cs]: Leave this using at the bottom.

Comment (DamienLaw) [reply src/Dommel/DommelMapper.Insert.cs]: Done. Visual Studio automatically sort `Using` statements for me upon saving.

**Comment by henkmollema** [comment]
Thanks again!

**Comment by henkmollema** [comment]
@DamienLaw just a small GitHub tip: you should associate your email address which you use to commit with your GitHub account to make sure all your commits belong to your user. You can add emails by going to GitHub -> Settings -> Emails.

File: src/Dommel/DommelMapper.Insert.cs (hunk 1 above)
Comment (henkmollema) [reply src/Dommel/DommelMapper.Insert.cs]: @DamienLaw you can configure this behavior in Visual Studio Options -> Text Editor -> C# -> Advances -> Place 'System' directives first when sorting usings.

**Comment by DamienLaw** [comment]
> @DamienLaw just a small GitHub tip: you should associate your email address which you use to commit with your GitHub account to make sure all your commits belong to your user. You can add emails by going to GitHub -> Settings -> Emails.

Thanks @henkmollema. I didn't realize that my global Git user settings are different than that of my GitHub account.

**Comment by DamienLaw** [comment]
> @DamienLaw just a small GitHub tip: you should associate your email address which you use to commit with your GitHub account to make sure all your commits belong to your user. You can add emails by going to GitHub -> Settings -> Emails.

Thanks @henkmollema. I didn't realize that my global Git user settings are different than that of my GitHub account.

**Pull request merged**